src/frontend/src/http.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct HttpOptions {
+    pub addr: String,
+    pub timeout: u64,
```

Comment:
Recommend using `humantime::Duration`, https://docs.rs/humantime/latest/humantime/struct.Duration.html

It's more readable in the configuration file.

Reply:
Yes, doing this using humantime-serde 1.1 together with the timeout test